email/1.3/modules/ROOT/edits/email-list.adoc

```diff
@@ -0,0 +1,210 @@
+= List Emails with Email Connector Examples - Mule 4
+:page-aliases: connectors::email/email-list.adoc
+
+Anypoint for Email (Email Connector) provides the *List - IMAP* and *List - POP3* operations to list emails from any part of the flow. The following examples illustrate how to configure these operations for the connector:
+
+* <<send-attachments,List Emails from an IMAPS Server>> +
+Configure the *List - IMAP* operation to list all the emails in the configured IMAP mailbox folder from an IMAPS server that has a TLS connection. Additionally, set a *For Each* scope component and two *Set Variable* components to iterate over the emails and save the email body and attachment content in variables.
+
+* <<send-attachments,List Emails from a POP3 Server>> +
+Configure the *List - POP3* operation to list all the emails in the configured POP3 mailbox folder that matched the criteria specified in the *Match with* parameter.
```

Comment:
This one is even trickier, because without making clear that these are UI LABELS, this should be...
```suggestion
Configure the List - POP3 operation to list all the emails in the configured POP3 mailbox folder that matched the criteria specified in the `Match with` parameter.
```
Maybe it would help to refer to the "font choices" entry in the style ref? Or if you do not think that that entry is clear, let me know?